server/src/main/java/io/deephaven/server/table/ops/TableServiceGrpcImpl.java

```diff
@@ -55,12 +59,17 @@
 import io.deephaven.server.session.SessionState;
 import io.deephaven.server.session.SessionState.ExportBuilder;
 import io.deephaven.server.session.TicketRouter;
+import io.deephaven.server.table.ops.GrpcTableOperation;
+import io.deephaven.time.DateTime;
 import io.deephaven.server.table.ExportedTableUpdateListener;
 import io.grpc.StatusRuntimeException;
 import io.grpc.stub.ServerCallStreamObserver;
 import io.grpc.stub.StreamObserver;
 
 import javax.inject.Inject;
+import java.lang.Object;
```

Comment:
Object should never need an import?

Reply:
Weird. Cleaned up and removed another unused import as well.